backend/experiment/actions/playback.py

```diff
@@ -49,6 +49,8 @@ def __init__(self,
                          for s in sections]
         if str(sections[0].filename).startswith('http'):
```

Comment:
I think this if condition is obsolete? The filename itself will not start with `http` anymore, right? Or did you leave this for backwards compatibility with playlists where the section csv has been altered?

Reply:
Yes, I left this in for backwards compatibility, but we should get rid of this.